MAUI/DataGrid/Grouping.md

```diff
@@ -485,4 +485,101 @@ The visibility of the grouped column can be customized by the [SfDataGrid.ShowCo
 this.dataGrid.ShowColumnWhenGrouped = false;
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+## Customize Group Icon with template
+
+By default , the SfDatagrid loads two different paths for denoting the group expanded and collapsed state. The group icon can be Customized by the [sfDataGrid.GroupExpandCollapseTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_GroupExpandCollapseTemplate) property. When loading as a template, you have the ability to load a single template that indicates the expanded state of a group. This template will then rotate to represent the collapsed state. Refer to the following code snippet for using template:
+
+{% tabs %}
+
+{% highlight xaml %}
+    <syncfusion:SfDataGrid VerticalOptions="FillAndExpand"
+                               HorizontalOptions="FillAndExpand"
+                               ItemsSource="{Binding OrderInfoCollection}"
+                               x:Name="dataGrid"
+                               GroupingMode="Multiple"
+                               AllowGroupExpandCollapse="True"
+                               AutoGenerateColumnsMode="None"
+                               >
+            <syncfusion:SfDataGrid.GroupExpandCollapseTemplate>
+                <DataTemplate>
+                    <Image Source="downward_icon.png"/>
+                </DataTemplate>
+            </syncfusion:SfDataGrid.GroupExpandCollapseTemplate>
+            
+            <syncfusion:SfDataGrid.GroupColumnDescriptions>
+                <syncfusion:GroupColumnDescription ColumnName="Name" />
+                <syncfusion:GroupColumnDescription ColumnName="ShipCity" />
+            </syncfusion:SfDataGrid.GroupColumnDescriptions>
+    </syncfusion:SfDataGrid>                           
+{% endhighlight %}
+
+{% highlight c# %}
+this.dataGrid.GroupingMode = GroupingMode.Multiple;
+this.dataGrid.AllowGroupExpandCollapse = true;
+dataGrid.GroupExpandCollapseTemplate = new DataTemplate(() =>
+        {
+            var imageView1 = new Image()
+            {
+                Source = "downward_icon.png",
+                Aspect = Aspect.AspectFit,
+            };
+            return imageView1;
+        });
+{% endhighlight %}
+{% endtabs %}
+
+![DataGrid with template](Images\Grouping\maui-datagrid-template.gif)
+
+## Customize Group Icon with template Selector
+
+By default , the SfDatagrid loads two different paths for denoting the group expanded and collapsed state. The group icon can be Customized by the [sfDataGrid.GroupExpandCollapseTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.SfDataGrid.html#Syncfusion_Maui_DataGrid_SfDataGrid_GroupExpandCollapseTemplate) property. When loading as a template Selector, you have the ability to load multiple templates that indicates the expanded and collapsed states of the groups. Refer to the following code snippet for using template selector: 
```

Comment:
When choosing a `GroupExpandCollapseTemplate` as a [DataTemplateSelector](https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.datatemplateselector?view=net-maui-7.0), you have the option to supply distinct templates for both the expanded and collapsed states of the group.

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/29c7b0b3-d9cd-43d5-b2d7-2b11c187eacd)